src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/ResourceLocator.js

```diff
@@ -19,8 +19,17 @@ class ResourceLocator extends React.Component<FieldTypeProps<?string>> {
     constructor(props: FieldTypeProps<?string>) {
         super(props);
 
-        const {dataPath, onChange, fieldTypeOptions, formInspector, value} = this.props;
-        const {generationUrl, modeResolver} = fieldTypeOptions;
+        const {
+            dataPath,
+            fieldTypeOptions: {
+                generationUrl,
+                modeResolver,
+                resourceStorePropertiesToRequest = {},
```

Comment:
Oh, and wouldn't it make more sene to call this `resourceStorePropertiesToGenerationRequest`? Then it would be clearer when this is used.

Reply:
I chose this name because it is already used in the `SingleSelection` and `Selection` field-type. but dont have a strong opinion

Reply:
Ok, I don't mind too much either... Was just a suggestion.